dotcom-rendering/src/web/components/ImageComponent.tsx

```diff
@@ -242,6 +242,27 @@ export const ImageComponent = ({
 
 	const palette = decidePalette(format);
 
+	const master = element.media.allImages.find(
+		(image) => image.fields.isMaster,
+	)?.url;
+
+	// Legacy images do not have a master so we fallback to the largest available
+	const descendingByWidth = (a: Image, b: Image) => {
+		return parseInt(b.fields.width) - parseInt(a.fields.width);
+	};
+	const fallbackImage =
```

Comment:
Do we want to generate a fallback image if we already have a `master`?This seems like unnecessary work for all modern images.

I would also favour `largestImage` rather than `fallbackImage`, to be more explicit.

```ts
const image = master ?? getLargestImage()
```